src/cloud-functions.ts

```
@@ -303,7 +304,9 @@ export function makeCloudFunction<EventData>({
           service,
         },
       });
-
+      if (opts.retry) {
+        trigger.eventTrigger.failurePolicy = { retry: {} };
```

Should this propagate the retry boolean instead of creating an empty object?

---

Nope - strangely enough, google cloud expects an empty object of type Retry here: https://cloud.google.com/functions/docs/reference/rest/v1/projects.locations.functions#EventTrigger

---

noted, interesting